### DISCUSSION TITLE
report of changes done

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3095] [Q&A]
Hi, 
Can we get a tabulated report of changes made by collaborators

**Comment by lucienicolier** [comment]
Hi,

You can look at all changes on QFieldCloud webUI, here is the doc [changes](https://docs.qfield.org/reference/qfieldcloud/projects/#changes)

I hope it helps

**Comment by MonikaShah** [comment]
Thanks
Can I have changes in form of csv or cls
Or some kind of tabulation

**Reply by suricactus** [reply]
no
